clash-cores/src/Clash/Cores/EbTb/Decoder.hs

```diff
@@ -0,0 +1,2070 @@
+-- |
+--   Copyright   :  (C) 2020-2021, Ben Sampson,
+--                      2024     , QBayLogic B.V.
+--   License     :  BSD2 (see the file LICENSE)
+--   Maintainer  :  QBayLogic B.V. <devops@qbaylogic.com>
+--
+--   8b/10b decoding look-up table
+module Clash.Cores.EbTb.Decoder where
+
+import Clash.Prelude
+
+-- https://github.com/bensampson5/libsv/blob/main/libsv/coders/decoder_8b10b.sv
```

Comment:
Please link to the commit hash that you used instead of `main`.
That way the link won't break when libsv changes things.
And if they change things we can diff things to see what changed and if we should copy any of those changes.

Reply:
I wanted to remove this, as 8b/10b is pretty much fixed anyway. I just copied this so I didn't have to write or generate the whole table myself. I did add the original author to the copyright notice.